pkg/operator/ceph/controller/pvc.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2021 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package controller
+
+import (
+	"context"
+
+	"github.com/rook/rook/pkg/clusterd"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ExpandPVCIfRequired will expand the PVC if requested size is greater than the actual size of existing PVC
+func ExpandPVCIfRequired(desiredPVC *v1.PersistentVolumeClaim, currentPVC *v1.PersistentVolumeClaim, clusterContext *clusterd.Context) {
```

Comment:
```suggestion
func ExpandPVCIfRequired(clusterContext *clusterd.Context, desiredPVC *v1.PersistentVolumeClaim, currentPVC *v1.PersistentVolumeClaim) {
```

Typically the context is the first argument in the signature.

Reply:
Please add unit tests to this too.

Reply:
Done